src/areaSearch/areaSearchSpecsPage.tsx

```diff
@@ -45,10 +46,13 @@ import AreaSearchMap from './components/AreaSearchMap';
 import { getInitialAreaSearchApplicationForm } from './helpers';
 import { ApplicationFormRoot } from '../application/types';
 import ScrollToTop from '../common/ScrollToTop';
+import { MultiPolygon } from 'geojson';
```

Comment:
I would recommend using `import type { MultiPolygon } from 'geojson';`
It is a bit more clear about the import, and it guarantees only type information is imported.